apps/chat/src/components/Chat/ChatHeader.tsx

```diff
@@ -178,6 +178,7 @@ export const ChatHeader = ({
                   tooltip={
                     <ChatInfoTooltip
                       model={model ?? conversation.model}
+                      version={model?.version}
```

Comment:
We already pass `model` with version here. You can read this property without additional property